mbuild/tests/test_compound.py

```diff
@@ -960,3 +960,27 @@ def test_from_pybel_monolayer(self):
         #            monolayer.unitcell.GetC()/10], 
         #        rtol=1e-3)
 
+    @pytest.mark.skipif(not has_matplotlib or not has_networkx, reason="Networkx and/or matplotlib not available")
+    def test_visualize_bond_graph_with_hexane(self, hexane):
+        import matplotlib
+        from matplotlib import pyplot as plt
+        import matplotlib.testing.compare as compare
+        from matplotlib.image import imread
+        import numpy as np
+        import random
+        matplotlib.testing.set_reproducibility_for_testing()
+        np.random.seed(42)
+        random.seed(42)
```

Comment:
Do we need both of these?

Reply:
According to `networkx` documentation 

>Sometimes you want more control over how the numbers are generated. In particular, you need to set the seed of the generator to make your results reproducible – either for scientific publication or for debugging. Both RNG packages have easy functions to set the seed to any integer, thus determining the subsequent generated values. Since this package (and many others) use both RNGs you may need to set the seed of both RNGs. Even if we strictly only used one of the RNGs, you may find yourself using another package that uses the other. Setting the state of the two global RNGs is as simple setting the seed of each RNG to an arbitrary integer.